### PR TITLE
Remove unused settings and associated code

### PR DESCRIPTION
**Pull request by AlanCoding**
##### SUMMARY
Trying to do some general cleanup.

##### ISSUE TYPE
 - Bug, Docs Fix or other nominal change

##### COMPONENT NAME
 - API

### CODE REVIEW

File: awx/settings/defaults.py [hunk 1]
```diff
@@ -808,11 +803,6 @@
 LOG_AGGREGATOR_RSYSLOGD_ERROR_LOG_FILE = '/var/log/tower/rsyslog.err'
 API_400_ERROR_LOG_FORMAT = 'status {status_code} received by user {user_name} attempting to access {url_path} from {remote_addr}'
 
-# The number of retry attempts for websocket session establishment
-# If you're encountering issues establishing websockets in a cluster,
-# raising this value can help
-CHANNEL_LAYER_RECEIVE_MAX_RETRY = 10
```
Comment (AlanCoding) [inline awx/settings/defaults.py:807]: If you follow the introduction of this setting, you can find that it was used internally. That use is no longer there.

File: awx/settings/defaults.py [hunk 2]
```diff
@@ -795,8 +792,6 @@
 INSIGHTS_SYSTEM_ID_FILE = '/etc/redhat-access-insights/machine-id'
 INSIGHTS_CERT_PATH = "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"
 
-TOWER_SETTINGS_MANIFEST = {}
```
Comment (AlanCoding) [inline awx/settings/defaults.py:791]: This pre-dates the configure-tower-in-tower feature as an early implementation of that same functionality. In fdcda43de6f214988a842073e7507e7c9c448810 you can see the intended use of the setting, which no longer exists.

File: awx/settings/defaults.py [hunk 3]
```diff
@@ -122,9 +122,6 @@
 # Absolute filesystem path to the directory to store logs
 LOG_ROOT = '/var/log/tower/'
 
-# The heartbeat file for the scheduler
-SCHEDULE_METADATA_LOCATION = os.path.join(BASE_DIR, '.tower_cycle')
```
Comment (AlanCoding) [inline awx/settings/defaults.py:118]: This was used for celery.

**Comment by AlanCoding** [comment]
ran some integration tests and it's looking good.

**Review by gamuniz — APPROVED** [review]

**Pull request merged**